oresat_dxwifi/templates/oresat_live.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <head>
+        <title>Oresat-Live Dxwifi Testpage</title>
+    </head>
+    <body>
+        <h1>State Options</h1>
+        <div>
+            <h2>Read</h2>
+            <button onclick="state_read()">Read State</button>
+            <p>State: <span id='state'></span></p>            
+            <h2>Write</h2>
+            <p>
+                OFF, BOOT, STANDBY, and ERROR probably shouldn't be written in practice as it doesn't really do anything. This is just for state testing.
+            </p>
+            <p>
+                The current decision is for filming and transmission to not occur simultaneously, so you must wait for one to finish (i.e. state becomes STANDBY) before writing one
+            </p>
+            <button onclick="state_write(State.OFF)">OFF</button>
+            <button onclick="state_write(State.BOOT)">BOOT</button>
+            <button onclick="state_write(State.STANDBY)">STANDBY</button>
+            <button onclick="state_write(State.FILMING)">FILMING</button>
+            <button onclick="state_write(State.TRANSMISSION)">TRANSMISSION</button>
+            <button onclick="state_write(State.ERROR)">ERROR</button>
+        </div>
+    </body>
+    <script>
+        const STATE_INDEX = 0x6002
```

Comment:
Make this a string. See the function documentation for `readValue` and `writeValue`: https://github.com/oresat/oresat-olaf/blob/master/olaf/_internals/rest_api/templates/_base.html